src/init/inst-functions.sh

```diff
@@ -31,16 +31,18 @@ InstallCommon()
 
   ./init/adduser.sh ${WAZUH_USER} ${WAZUH_GROUP} ${INSTALLDIR}
 
+  # Folder for temporary files
+  ${INSTALL} -d -m 0660 -o ${WAZUH_USER} -g ${WAZUH_GROUP} ${INSTALLDIR}tmp/wazuh-server
```

Comment:
We need to ask the Cpp team about the `vd_1.0.0_vd_4.10.0.tar.xz` being generated here and is making us create the folder. It may be needed to put it in another folder.